apps/calendar/js/bridge.js

```diff
@@ -91,10 +94,70 @@ var persistEvent = co.wrap(function *(event, action, capability) {
   return provider[action + 'Event'](event.data);
 });
 
-exports.getSetting = co.wrap(function *(id) {
-  var settingsStore = core.storeFactory.get('Setting');
-  return yield settingsStore.getValue(id);
-});
+exports.getSetting = function(id) {
```

Comment:
I think one way to look at it is there are two inputs which are the default and whatever the user has currently chosen and those two things should reactively give the select value (and don't need `getSetting`)